CONTRIBUTING.md

```diff
@@ -210,6 +210,12 @@ Per the
 [API guidelines](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter),
 `get_()` prefixes are discouraged.
 
+#### Don't elide generic lifetimes
+
+We prefer not to elide lifetimes when naming types that are generic over
+liftimes. Always include a lifetime placeholder (e.g. `<'_>`) to avoid
```

Comment:
"liftimes" -> "lifetimes"

Reply:
Fixed thanks. Turns out writing a docs update at the very end of the day isn't a great way to avoid typos :sweat: